source/signature_subpacket/embedded.cpp

```diff
@@ -46,6 +46,8 @@ namespace pgp::signature_subpacket {
     /**
      *  Comparison operators
      *
+     *  Note: this function is UB when used with a moved-from object.
```

Comment:
I'd explicitly use the word `precondition`, since this is a well-known term which people may be searching for

Reply:
Fair enough. Somewhat related: should we continue to duplicate the documentation in both the header and the implementation? That's bound to go wrong at some point.